### PR TITLE
Fix filter normalization for materials

### LINKED ISSUE #226: Materials Project Training
Hello, I want to eventually train SchNet models on some crystal structures that I have obtained using Quantum ESPRESSO, and I figured I would first test out my model settings on the Materials Project database. While training the formation energy per atom, the MAE seems to plateau at 0.6 eV/atom, which is quite a bit worse than the 0.041 eV/atom that I see in the SchNetPack paper: https://doi.org/10.1021/acs.jctc.8b00908 I have the code I used to construct the model below. Are the settings I used reasonable and similar for the ones used in the benchmark, or is there something I'm missing? 
```
mp = MaterialsProject("./mp.db", load_only=[MaterialsProject.EformationPerAtom], environment_provider=sch.environment.AseEnvironmentProvider(5))

train, val, test = spk.train_test_split(
        data=mp,
        num_train=int(4/5*len(mp)),
        num_val=int(1/5*len(mp)),
        split_file="split.npz"
    )

train_loader = spk.AtomsLoader(train, batch_size=16, shuffle=True)
val_loader = spk.AtomsLoader(val, batch_size=16)

schnet = spk.representation.SchNet(
    n_atom_basis=128,
    n_filters=128,
    n_gaussians=30,
    n_interactions=6,
    cutoff=5., cutoff_network=spk.nn.cutoff.CosineCutoff
)

output = spk.atomistic.Atomwise(n_in=128, property=MaterialsProject.EformationPerAtom)
model = spk.AtomisticModel(representation=schnet, output_modules=output)
optimizer = Adam(model.parameters(), lr=0.0001)

loss = trn.build_mse_loss([MaterialsProject.EformationPerAtom])

metrics = [spk.metrics.MeanAbsoluteError(MaterialsProject.EformationPerAtom)]

hooks = [
    trn.CSVHook(log_path='./', metrics=metrics),
        trn.ReduceLROnPlateauHook(
        optimizer,
        patience=5, factor=0.8, min_lr=1e-6,
        stop_after_min=True
    )
]

trainer = trn.Trainer(
    model_path='./',
    model=model,
    hooks=hooks,
    loss_fn=loss,
    optimizer=optimizer,
    train_loader=train_loader,
    validation_loader=val_loader,
)
```

### PR DESCRIPTION
**Pull request by ktschuett**
Fixes #226 

Parsing had a bug where the filter were not normalized for Materials Project and OMDB.

Still needs to be tested.

### CODE REVIEW

**Comment by jhrmnn** [comment]
Already running it, will report back.

**Comment by ktschuett** [comment]
There seem to be still some differences to the old version...

**Comment by jhrmnn** [comment]
So the materials dataset should now run fine?

**Comment by ktschuett** [comment]
@jhrmnn We checked a lot of things already. All other datasets still give the good results and the old scripts give the same results as the new ones.  Multiple theories:
1. The MP introduced a data deprecation scheme. There could be still different data. There is a flag to load deprecated structures.
2. Some bug when loading the data with pymatgen or other changes in 3rd party libraries (e.g. ASE).
3. The settings given in the paper are wrong and we used shorter patience, larger learning rate, more features, etc.
4. We're all going crazy.

Stefaan is still checking some of these things. Meanwhile, I started implementing the new generation of schnetpack and will test whether it magically works again, once I'm done :-)

**Comment by jhrmnn** [comment]
Alright, no pressure :) Please keep me posted.

**Comment by MrRobot2211** [comment]
Hi are there any updates on this?

**Comment by GravityPHY** [comment]
> I think the most likely reason for the different performance is that the Materials Project replaced some old data with new calculations and larger structures were added. We are almost done implementing a major update for SchNetPack, then we will try again.

If this is the case, will delete the data of large structures (e.g. num_atoms>200) work out as before?

**Comment by jhrmnn** [comment]
#256 enabled reconstructing the Materials Project dataset at a given timestamp. One can reconstrcut the dataset with precisely the same number of samples as reported in the original SchNet JCP paper.

This doesn’t solve the issue of older data having been replaced, but hopefully that should be negligible.
